source/quick-reference.txt

```diff
@@ -28,13 +28,13 @@ their related reference and API documentation.
           .. input::
              :language: go
 
-             err = coll.FindOne(context.TODO(), bson.D{{"rating", 5}}).Decode(&result)
+             err = coll.FindOne(context.TODO(), bson.D{{"xValue", 5}}).Decode(&result)
```

Comment:
S: I think that this example is still a little difficult to follow. I noticed that elsewhere on this page you used sample documents that includes names -- maybe you could also use that here with an age filter or something!

Suggestion applies to all examples that use this sample data